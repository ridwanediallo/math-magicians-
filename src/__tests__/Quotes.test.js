import renderer from 'react-test-renderer';
import Quote from '../components/Quotes';

it('Displays the quote correctly', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
