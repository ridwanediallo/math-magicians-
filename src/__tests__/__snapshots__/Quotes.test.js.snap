// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Displays the quote correctly 1`] = `
<div
  className="container"
>
  <p
    className="quote"
  >
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. -William Paul Thurston
  </p>
</div>
`;
